packages/sdk/src/client/backend.ts

```diff
@@ -91,6 +89,55 @@ export class MetromApiClient {
         });
     }
 
+    async fetchCampaign(params: FetchCampaignParams): Promise<Campaign> {
+        const url = new URL("campaigns", this.baseUrl);
```

Comment:
Here the base url on the API is `campaign` without the `s`, but it probably makes more sense to have the `s` and thus change the API path on the backend, wdyt?